plugins/tracers/gstinterlatency.c

```diff
@@ -173,6 +173,7 @@ calculate_latency (GstInterLatencyTracer * interlatency_tracer,
 
     if (GST_IS_EVENT (ev))
```

Comment:
You need brackets here to avoid unref ev in case GST_IS_EVENT(ev) fails

Reply:
Good call, addressed in 7011da8e6cb818c24a72cc4ff8025df443778ecc